plugins/genocide/genocide.cpp

```diff
@@ -16,29 +16,47 @@ class genocide : public bz_Plugin
     virtual void Cleanup ( void );
     virtual void Event ( bz_EventData *eventData );
 
-    bool allowSuicide = false;
-    bool rogueGenocide = false;
+    bool disableSuicide = false;
+    bool rogueAsTeam = false;
 };
 
 BZ_PLUGIN(genocide)
 
-void genocide::Init ( const char* commandLine )
+void genocide::Init ( const char* cmdLine )
 {
     bz_debugMessage(4, "genocide plugin loaded");
-    if (commandLine)
+    if (strlen(cmdLine) != 0)
     {
-        int conf = atoi(commandLine);
-        if (conf == 1 || conf == 2)
-            rogueGenocide = true;
-        if (conf == 2)
-            allowSuicide = true;
+        if (strcmp(cmdLine, "rogueAsTeam,disableSuicide") == 0)
+        {
+            rogueAsTeam = true;
+            disableSuicide = true;
+        }
+        else if (strcmp(cmdLine, "disableSuicide,rogueAsTeam") == 0)
+        {
+            disableSuicide = true;
+            rogueAsTeam = true;
+        }
+        else if (strcmp(cmdLine, "rogueAsTeam") == 0)
+        {
+            rogueAsTeam = true;
+        }
+        else if  (strcmp(cmdLine, "disableSuicide") == 0)
+        {
+            disableSuicide = true;
+        }
+        else
+        {
+            bz_debugMessage(0, "Error: Check arguments.");
+        }
+        
     }
 
-    if (rogueGenocide == true)
-        bz_debugMessage(4, "rogue genocide enabled");
+    if (rogueAsTeam == true)
```

Comment:
just a personal preference, but testing boolean variables against explicit boolean values is redundant.

Well named variables should be self-explanatory, as in
```c++
if (rogueAsTeam)
   ...
```